react/features/film-strip/components/FilmStrip.js

```diff
@@ -112,7 +113,15 @@ function _mapStateToProps(state) {
          * @private
          * @type {Participant[]}
          */
-        _participants: state['features/base/participants']
+        _participants: state['features/base/participants'],
+
+        /**
+         * The indicator which determines whether toolbar is visible.
+         *
+         * @private
+         * @type {boolean}
+         */
+        _visible: !state['features/toolbar'].visible
```

Comment:
Because the React Component FilmStrip is used on mobile only at the time of this writing and on mobile the film strip is visible when the toolbar is not.